assets/javascripts/modules/responsiveInlineDetails.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function() {
+
+  $('[data-responsive-inline-details]').each(function() {
+    var $detailsHolder,
+    $detailsTargets,
+    $detailsButton;
+
+    $detailsHolder = $(this);
+    $detailsButton = $detailsHolder.find('.inline-details-toggle');
+    $detailsTargets = $('.' + $detailsHolder.data('responsive-inline-details'));
```

Comment:
Wouldn't it be easier to have `data-responsive-inline-details` to store actual jQuery selector to the details target? That would give much more flexibility to that selector as you could specify ID, class/classes or other selectors.